CONTRIBUTING.md

```diff
@@ -130,19 +151,27 @@ $ make build
 $ <Iterate your development>
 $ git push origin <your_branch_for_new_pr>
 ```
+**Tests for your changes**
 
-1. Add unit tests for new functionality. Add e2e tests for major changes to functionality.
-1. If you don't have a live object store ready, you can use the `make test-local` command.
-
-NOTE: this command skips tests against live object storage systems by specifying environment variables; this causes the
+* Add unit tests for new functionality. Add e2e tests for major changes to functionality.
+* If you don't have a live object store ready, you can use the `make test-local` command.
```

Comment:
There is overlap with #Running tests.

Should we link to running test and copy all overlap there? 🤗

Reply:
I think, lets move the content to `#Running tests`, would be nice to have all content for testing in one place. WDYT?